activity.py

```diff
@@ -579,7 +579,7 @@ def add_value(self, label, value):
         elif selected:
             path = int(str(self.model.get_path(selected))) + 1
         try:
-            _iter = self.model.insert(path, [label, value])
+            _iter = self.model.insert(path, [label, str(value)])
```

Comment:
Are you sure it should be changed to str here, instead of TypeError Exception handling? Also I suggest you mention just the error and some part of the traceback in the commit message than copy pasting the entire traceback :)

Reply:
Yeah actually in https://github.com/sugarlabs/AnalyzeJournal/pull/14/files#diff-03091df2633029b2c2893b03883f5296R542 , it is initialised with str. We can try Exception handling if it is not reproducible by your test system. Can you tell about your test system?

Reply:
I agree with @JuiP. The exception handling does take care of the typecast that has been introduced in the try block